src/gatsby/models/markdown-content/index.js

```diff
@@ -0,0 +1,40 @@
+const buildApi = require('../../utils/models')
+
+/*
```

Comment:
joint comments into one blog, try to explain it a little bit better what's happening here